app/liquid_tags/spotify_tag.rb

```diff
@@ -10,23 +11,18 @@ class SpotifyTag < LiquidTagBase
 
   def initialize(tag_name, uri, tokens)
     super
-    @parsed_uri = parse_uri(uri)
+    @parsed_uri = generate_embed_link(parse_uri(uri))
```

Comment:
Ok I thought there was something wrong 😉 I was working on a PR for #2739 but the test kept failing.

Reply:
But now that I think of it this would mean that the Spotify liquid tag was broken in the first place. Or am I wrong?

Reply:
`generate_embed_link` was being called inline in the html before so I had to move it inside the `initialize` method instead. Just checked master branch and the Spotify tag works. Styling is off though and the `iframe` needs more height to access the play button. 

Are you working on the PR off of this branch or master?

Reply:
I was working/testing on the current master branch but am now waiting till this PR is merged.
The current master branch spec was giving me an error (for the valid url) locally.

Reply:
Ahh that's odd. I just ran `rspec` on the `spotify_tag_spec.rb` and its tests passed. 😕Perhaps another spec is giving an error?

Reply:
@maestromac pointed out `spotify_tag_spec.rb` just expects no errors but doesn't test if the tag actually works. That needs to change.

Reply:
Might just be me then... `rspec` always seems to give me errors but then when I make a PR and Travis runs the tests they all pass.
Don't know what I am doing wrong, doesn't help that I am not used to working in Ruby and certainly not rpec.

Reply:
If you wanna shoot me a message on [DEV](dev.to/mariocsee), I can try to help you out and get a better picture of what's going on!